CalibTracker/SiStripChannelGain/plugins/SiStripGainFromCalibTree.cc

```diff
@@ -479,7 +480,25 @@ void SiStripGainFromCalibTree::bookDQMHistos(const char* dqm_dir, const char* ta
 
     int elepos = (m_harvestingMode && AlgoMode=="PCL")? Harvest : statCollectionFromMode(tag);
 
-    Charge_Vs_Index[elepos]           = dbe->book2S(cvi.c_str()     , cvi.c_str()     , 88625, 0   , 88624,2000,0,4000);
+    //setting up the bin array for the Charge_Vs_Index histogram
+    float* binXarray = new float[NStripAPVs+1];
+    for(int a=0;a<=NStripAPVs;a++){
+       binXarray[a] = (float)a;
+    }
+ 
+    float* binYarray = new float[688];
+    double p0 = 5.445;
+    double p1 = 0.002113;
+    double p2 = 69.01576;
```

Comment:
@mmusich concerning the place of the "magic numbers". These numbers concerns the binning of histograms which - for the time being - are duplicated among the PCWorker and the SiStripGainFromCalibTree. The re-engineering of SiStripGainFromCalibTree.cc will happen later and is not a subject of this PR.